schema/profile.js

```diff
@@ -281,7 +281,7 @@ class Profile extends BaseModel {
           to: 'pils.profileId'
         },
         filter: f => {
-          return f.skipUndefined().where('establishmentId', f.context().establishmentId).orderBy('issueDate', 'desc');
+          return f.skipUndefined().orderBy('issueDate', 'desc');
```

Comment:
You don't need the `skipUndefined` because there's no subsequent `where` clause which might be undefined.

Reply:
derp